docs/user/security/authorization/index.asciidoc

```diff
@@ -68,6 +68,8 @@ image::user/security/images/create-role-index-example.png[Create role with index
 [[index_privilege_dls_example]]
 ===== Example: Grant read access to specific documents in indices that match the `filebeat-*` pattern
 
+{ref}/document-level-security.html[Document-level security] is a https://www.elastic.co/subscriptions[subscription feature].
```

Comment:
Fun fact: the Subscriptions page says "Document-level security", but the ES docs say "Document level security". The former sounds more correct to me so I went with it.

Reply:
You're right.  Document-level security.